spec/controllers/journal_entries_controller_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+RSpec.describe JournalEntriesController, type: :controller do
+  let(:journal) do
+    FactoryBot.create(:journal)
+  end
```

Comment:
`let(:journal) { FactoryBot.create(:journal) }`

we will use do block only if line limit (110 char) exceeds.